components/layout/construct.rs

```diff
@@ -1316,6 +1316,39 @@ impl<'a, ConcreteThreadSafeLayoutNode: ThreadSafeLayoutNode>
         self.build_flow_for_block_like(flow, node)
     }
 
+    /// Builds a fragment for a node with 'display: inline-flex'.
+    // Shared code with build_fragment_for_inline_block
+    fn build_fragment_for_inline_flex(&mut self, node: &ConcreteThreadSafeLayoutNode)
```

Comment:
Also, small nit: the `Shared code` comment needs another slash in front of it to be a doc comment.